### ISSUE TITLE
Soundswitch 3.1 ?

### ISSUE THREAD

**Issue by k8six** [Belphemur/SoundSwitch#213]
[//]: # (Please provide us your logs/configuration folder if you are reporting a crash.)
[//]: # (Compress the whole %AppData%\SoundSwitch directory and upload it.)

[AudioEndPointLibrary.zip](https://github.com/Belphemur/SoundSwitch/files/1281015/AudioEndPointLibrary.zip)

Sorry I have NO clue where the logs hide. I looked in Event Viewer to no avail. 

My problem is Soundswitch is running BUT it is not in the taskbar. If I go to Task Manager it is there but not on taskbar. When it launches I get the message, " Impossible to Register Hotkey:Alt + Control + F11:True" 
If I hit Alt, Ctrl, F11 nothing happens. 
This issue did not start until I upgraded to a i7 and had to re register Windows 10 Pro x  64. 
I did a uninstall, reboot and reinstalled with same results. Any ideas?

Since I first wrote this I Uninstalled Soundswitch and went into the registry and deleted all instances of the program. Rebooted and reinstalled with same results. It does not show up on taskbar even though it IS running. I cannot access program at all. Am running version v3.15.1.15427.

**Comment by k8six** [comment]
OK I went to APP DATA and deleted SoundSwitch and then relaunched the program. I set everything back up yet the NORMAL Soundswitch icon does NOT appear in the taskbar. I moved the mouse over the speaker and noticed it said Soundswitch and come to find out it acts just like the normal Soundswitch Icon. If I right click I can go into setting, etc. If I left click I can select one of my 4 devices. SO basically it work BUT why don't I get the Normal Soundswitch icon instead of a speaker?
